lib/client.js

```diff
@@ -99,6 +99,7 @@ Client.prototype._initializeServices = function(endpoint) {
 
 Client.prototype._initializeOptions = function(options) {
   this.wsdl.options.attributesKey = options.attributesKey || 'attributes';
+  this.wsdl.options.forceSoap12 = typeof options.forceSoap12 === 'undefined' ? false : options.forceSoap12;
```

Comment:
can you change this to `this.wsdl.options.forceSoap12 = !!options.forceSoap12;`?